### PR TITLE
feat(alicloud): Add tag image 

### PR DESCRIPTION
**Pull request by 956081712**
add find image  and tag image

### CODE REVIEW

**Comment by 956081712** [comment]
Excuse me, when can I merge

**Comment by 956081712** [comment]
Excuse me, when can I merge

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AliCloudImageFinder implements ImageFinder {
+
+  final String cloudProvider = "alicloud";
+
+  @Autowired OortService oortService;
```
Comment (marchello2000) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java]: we prefer @Autowired constructors over @Autowired fields

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    /*
+     * Find the most recently created image.
+     * (optimized for readability over efficiency given the generally small # of images)
+     */
+    if (allMatchedImages.size() > 0) {
+      AliCloudImage latestImage = allMatchedImages.get(0);
+      return Collections.singletonList(latestImage.toAliCloudImageDetails());
+    } else {
+      return null;
+    }
```
Comment (marchello2000) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java]: nit: else not needed

```suggestion
    }
    
    return null;
```

**Comment by marchello2000** [comment]
Please update the commit message format as requested then we can merge it

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java (hunk 1 above)
Comment (956081712) [reply orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java]: yourn meacn is 
  private final OortService oortService;

  private final ObjectMapper objectMapper;

  @Autowired
  public OracleImageFinder(OortService oortService, ObjectMapper objectMapper) {
    this.oortService = oortService;
    this.objectMapper = objectMapper;
  }

?

Comment (956081712) [reply orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java]: yourn mean is
private final OortService oortService;

private final ObjectMapper objectMapper;

@Autowired
public AliCloudImageFinder(OortService oortService, ObjectMapper objectMapper) {
this.oortService = oortService;
this.objectMapper = objectMapper;
}

?

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java (hunk 2 above)
Comment (956081712) [reply orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java]: tank you

Comment (956081712) [reply orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/alicloud/AliCloudImageFinder.java]: thank you

**Comment by 956081712** [comment]
Excuse me, when can I merge

**Comment by 956081712** [comment]
Excuse me, when can I merge

**Pull request merged**